atom/browser/api/atom_api_system_preferences.h

```diff
@@ -9,6 +9,7 @@
 
 #include "atom/browser/api/event_emitter.h"
 #include "base/callback.h"
+#include "base/values.h"
```

Comment:
That's what I though, but for some reason, the CI is failing if we don't include this header :/
https://travis-ci.org/electron/electron/jobs/150172395